extensions/omni_python/docs/intro.md

```diff
@@ -0,0 +1,272 @@
+
+Load the filesystem files.
+```postgresql
+select omni_schema.load_from_fs(demo_function());
+```
+
+!!! tip "Optional Tip"
+
+    You can set a reload command for reloading the filesystem changes.
+    ```postgresql
+    \set reload 'select omni_schema.load_from_fs(demo_function());'
+    ```
+
+
+Run Omnigres in a Docker container. Make sure to mount the local directory as a volume on the correct path.
+```shell
+docker run --name omnigres \
+           -e POSTGRES_PASSWORD=omnigres \
+           -e POSTGRES_USER=omnigres \
+           -e POSTGRES_DB=omnigres \
+           --mount source=omnigres,target=/var/lib/postgresql/data -v $(pwd)/python-files:/python-files \
+           -p 127.0.0.1:5433:5432 --rm ghcr.io/omnigres/omnigres-slim:latest
+```
+
+Let's try it out!
+```postgresql
+omnigres=# select hello();
+   hello
+------------
+ Hey there!
+```
+```postgresql
+omnigres=# select add(5, 10);
+ add
+-----
+  15
+```
+```postgresql
+omnigres=# select subtract(5, 10);
+ subtract
+----------
+       -5
+```
+
+## Flask
+
+For Flask framework integration, we have a few more steps.
+
+Create `omni_httpd` extension to be able to handle HTTP requests.
+```postgresql
+create extension if not exists omni_httpd cascade;
+```
+
+Add the following in `requirements.txt` file.
+```
+omni_http[Flask]
+```
+
+Create HTTP listeners for our Flask application. For example, let's add a HTTP listener for port 5000.
+```postgresql
+with
+    listener as (insert into omni_httpd.listeners (address, port) values ('0.0.0.0', 5000) returning id),
+    handler as (insert into omni_httpd.handlers (query) values
+                                                 (
+                                                 $$ select handle(request.*) from request$$
+       ) returning id)
+insert
+into
+    omni_httpd.listeners_handlers (listener_id, handler_id)
+select
+    listener.id,
+    handler.id
+from
+    listener,
+    handler;
+```
+
+Let's say you have a table called `employees`.
+```postgresql
+create table employees (
+   id uuid,
+   name text not null,
+   department text not null,
+   salary integer not null
+);
+```
+
+Now you can update your Python files (in the mounted volume) to include Flask functionality. For example, you can define endpoints to fetch list of all employees, fetch a particular employee, as well as create a new employee record.
+``` py hl_lines="26-30"
+from omni_python import pg
+from omni_http import omni_httpd
+from omni_http.omni_httpd import flask
+from flask import Flask, jsonify, make_response
+import uuid
+
+app = Flask('myapp')
+
+def result_to_json(result):
+    rows = []
+    for row in result:
+        rows.append(dict(row))
+    return json.dumps(rows)
+
+@app.route('/employees', methods=['POST'])
+def create_employee():
+    from flask import make_response, request
+    json_data = json.loads(request.data.decode('UTF-8'))
+
+    employee_id = uuid.uuid4()
```

Comment:
if the `uuid` is replaced with a traditional `int8` that is generated, no need in this.